### PR TITLE
[Cairo-SR3]Change Lombok' version to 1.18.2 #585 [Dozer 6.4.1]Upgrade Dozer #586

### PR DESCRIPTION
**Pull request by btkousil**
Please review #585 #586.

### CODE REVIEW

**Comment by btkousil** [comment]
@btyanagawamg 
Please review

File: pom.xml [hunk 1]
```diff
@@ -267,7 +267,11 @@
         <version>${selenium.version}</version>
       </dependency>
       <!-- == End Test == -->
-
+      <dependency>
```
Comment (bttanakaysd) [inline pom.xml]: - [ ] You should write that we are specifying a version for workaround.

Comment (btyanagawamg) [reply pom.xml]: @bttanakaysd 
42d495bc3bc8cdafb898bc4f9d99817e3e2b4a47 
I added comment why we specify version .

Comment (btyanagawamg) [reply pom.xml]: @bttanakaysd 
949cd29ef8b26fb87b916cd2cf163a7d1d2361e4 
I modified comment to smart .

**Pull request merged**